### PR TITLE
Add cold staking setup information

### PR DESCRIPTION
**Pull request by thecrypt0hunter**
I have created a markdown file describing the process for setting up cold staking.

### CODE REVIEW

File: Documentation/cold-staking-setup.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+## COLD STAKING SETUP INSTRUCTIONS
+
+**What you will need** 
+
+ - **Hot wallet** - this will be used for staking online, you can skip this if you are using _www.trustaking.com_
+ - **Cold wallet** - this will hold your funds offline
+   
+_Please note. It's important that both wallets are running on separate full nodes._
```
Comment (quantumagi) [inline Documentation/cold-staking-setup.md]: It's important that both wallets are used with separate full nodes instances, and are not present in the same data folder.

File: Documentation/cold-staking-setup.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ 
+ ---
+****Cold wallet****
+
+_If you are using www.trustaking.com, we provide a command to simplify this process._
+
+1. Fund the Cold wallet with coins that you eventually want to stake, these coins should go into the standard "account 0".
+2. Convert your Cold wallet to be enabled for cold staking by using the `cold-staking-account` API method with "isColdWalletAccount" set to "true"
+3. Get your Cold wallets coldstakingcoldaddress using by using the `cold-staking-address` API method with "IsColdWalletAddress" set to "true"
+4. Then, call the `setup-cold-staking` API to build the transaction and send coins from the Hot Wallet Address (Step #2) to Cold Wallet Address (Step #6). This will return the hex that you use in the next step.
```
Comment (quantumagi) [inline Documentation/cold-staking-setup.md]: Clarify where step #2 and step #6 are located.

File: Documentation/cold-staking-setup.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+1. Convert your Hot wallet to be enable cold staking by using the the `cold-staking-account` API method with isColdWalletAccount set to "false"
+2. Get your Hot wallets coldstakinghotaddress using by using the `cold-staking-address` API method with "IsColdWalletAddress" set to "false"
+3. Then start the node staking with the Hot wallet from the command line or config file.
+ 
+ ---
+****Cold wallet****
+
+_If you are using www.trustaking.com, we provide a command to simplify this process._
+
+1. Fund the Cold wallet with coins that you eventually want to stake, these coins should go into the standard "account 0".
```
Comment (quantumagi) [inline Documentation/cold-staking-setup.md]: In this case (see below) you seem to be funding the setup transaction from the hot wallet, so perhaps this should read "Fund the Hot wallet with coins..."

**Comment by rowandh** [comment]
This just seems like a not-so-subtle ad for Trustaking. Please remove all the references to it, they do not belong in this document. Thanks for the contribution to the rest of the docs.

File: Documentation/cold-staking-setup.md (hunk 1 above)
Comment (thecrypt0hunter) [reply Documentation/cold-staking-setup.md]: Updated

File: Documentation/cold-staking-setup.md (hunk 2 above)
Comment (thecrypt0hunter) [reply Documentation/cold-staking-setup.md]: Clarified

File: Documentation/cold-staking-setup.md (hunk 3 above)
Comment (thecrypt0hunter) [reply Documentation/cold-staking-setup.md]: Updated to reflect that the funding is from the cold wallet

**Comment by thecrypt0hunter** [comment]
I think I have made all the requested changes; please review

**Comment by thecrypt0hunter** [comment]
Actually, there seems to be a styling issue ... I'll take a look at it later.

**Comment by thecrypt0hunter** [comment]
@quantumagi & @rowandh - thanks for reviewing ... I have made the requested changes. Let me know if there are any other comments.

**Pull request merged**